modules/face/samples/sampleKazemi.cpp

```diff
@@ -0,0 +1,106 @@
+#include<iostream>
+#include<vector>
+#include<fstream>
+#include "opencv2/face/face_alignment.hpp"
+#include "opencv2/core.hpp"
+#include "opencv2/highgui.hpp"
+#include "opencv2/imgproc.hpp"
+#include "opencv2/imgcodecs.hpp"
+using namespace std;
+using namespace cv;
+int main(int argc,char** argv){
+    //Give the file containing the filenames of filenames as argument
+    if(argc!=2){
+        cout<<"No file found";
+        return 0;
+    }
+    string file = argv[1];
+    //cout<<file<<endl;
+    ifstream f(file.c_str());
+    if(!f.is_open()){
+        cout<<"Error while loading file";
+        return 0;
+    }
+    //create a vector to store names of files in which annotations
+    // and image names are found
+    /*The format of the file containing annotations should be of following format
+        /data/abc.jpg
+        123.45,345.65
+        321.67,543.89
+     */
+    vector<string> filenames;
+    string s;
+    while(getline(f,s)){
+        //cout<<s<<endl;
+        filenames.push_back(s);
+    }
+    f.close();
+    //create a vector to store image names
+    vector<string> imagenames;
+    //create object to get landmarks
+    vector<vector<Point2f> > trainlandmarks;
```

Comment:
In general, for visual clarity I suggest that if you have `vector<vector<type> >` you change it to `vector< vector<type> >`. The extra space gives some balance and if a user would make an overload for `<<` then that would not form an issue in the code.